src/shared/infrastructure/profile-metrics/profile-metrics.json

```
@@ -78,28 +78,28 @@
   },
   "mutex:delay:nanoseconds:contentions:count": {
     "id": "mutex:delay:nanoseconds:contentions:count",
-    "description": "Total time spent waiting due to mutex contentions",
+    "description": "Time spent waiting due to mutex contentions",
     "type": "delay",
     "group": "mutex",
     "unit": "ns"
   },
   "process_cpu:alloc_samples:count:cpu:nanoseconds": {
     "id": "process_cpu:alloc_samples:count:cpu:nanoseconds",
-    "description": "Count of memory allocation samples during CPU time",
+    "description": "Number of memory allocation samples during CPU time",
     "type": "alloc_samples",
     "group": "memory",
     "unit": "short"
   },
   "process_cpu:alloc_size:bytes:cpu:nanoseconds": {
     "id": "process_cpu:alloc_size:bytes:cpu:nanoseconds",
-    "description": "Total bytes allocated during CPU time",
+    "description": "Size of memory allocated during CPU time",
     "type": "alloc_size",
     "group": "alloc_size",
     "unit": "bytes"
   },
   "process_cpu:cpu:nanoseconds:cpu:nanoseconds": {
     "id": "process_cpu:cpu:nanoseconds:cpu:nanoseconds",
-    "description": "Total nanoseconds of CPU time consumed",
+    "description": "Time of CPU consumed",
```

Do we need to state that the time is in nanoseconds? also, does the updated wording have the same meaning as the original? What about this?
```suggestion
    "description": "CPU time consumed (in nanoseconds)",
```